classes/01class/exercises/README.md

```diff
@@ -0,0 +1,21 @@
+# Labs and Exercises
```

Comment:
Hi Matt. This seems to be an older exercises/README.md and should not be submitted. 

I think you'll need to merge the master branch in your dev branch to update your local repository (check out the Git class for `git merge`). Let me know if you have problems.

Also, please use the existing `exercises` folder (lowercase).

Thanks!

Reply:
Hey Denis!
Thanks for pointing that out, not sure how I ended up pushing the README.
I think it was because I was in the root exercise location instead of pushing from the folder itself.
Although I seem to be having issues merging to the remote/master.
I'm getting already up to date

 git fetch origin master; git merge origin/master
Enter passphrase for key '/root/.ssh/id_rsa': 
From github.com:starkmatt/academy
 * branch            master     -> FETCH_HEAD
Already up to date.

But still having issues!

Thanks!